### ISSUE TITLE
Site Footer

### ISSUE THREAD

**Issue by alex-418** [BCcampus/bcc-sage#154]
## Submit a feature request or bug report

- [x] This is a feature request
- [ ] This is a bug report
- [ ] This request isn't a duplicate of an [existing issue](https://github.com/BCcampus/bcc-sage/issues)
- [ ] You cleared your Web Browsers cache to ensure all new styling and images were loaded. 

## Feature Request

**Please provide use cases for changing the current behavior:**

Footer includes 3 columns, where widgets can be placed.

**Comment by bdolor** [comment]
blocked by https://github.com/BCcampus/bcc-sage/issues/159

**Comment by bdolor** [comment]
resolved via https://github.com/BCcampus/bcc-sage/pull/171
note: opted to use font awesome icons and there is no kaltura icon available. PNG Icons supplied by designer are not the right size.

**Comment by murphybarb** [comment]
Provide the required size of the icons and I will request them from the designer. 

Are the items under "Quick Links" editable in the dashboard or a dev task? If so, the links to appear there are incorrect. 

Are the links in  " Newsletter", "News Archives", "Careers" and "What we do" editable in the dashboard or a dev task? If so, they are also incorrect. 

I can provide the correct text and links here in the issue, if required.

**Comment by bdolor** [comment]
@murphybarb - the "Quick Links" are editable menus, as are the "Newsletter" and "News Archives". 

I could make a guess as to what the correct size is, but would rather leave that up to the designer. Right now they are 60x60px

**Comment by murphybarb** [comment]
Hey @bdolor, you had indicated PNG Icons supplied by designer were not the right size. So assumed you had a specific size that was required? 

I'm looking in the [assests](https://cloud.bccampus.ca/index.php/apps/files/?dir=/*Current%20Jobs/283%20Web%20Consolidation%20Project/Visual%20Design/BCcampus.ca/BCcampus.ca%20%E2%80%93%C2%A0Assests%20for%20DevOps/Image%20assets&fileid=2957189) folder and see everything you may have needed for the icons may not have been provided. Is that correct? 

I will ask the designer to provide all the required icon graphics and information you require. 

Also, in the footer, at the very bottom, the text " Newsletter", "News Archives", "Careers" and "What we do" are incorrect. They should be: 
"Terms & Conditions | Privacy Policy | © BCcampus"

With these links: 
Terms & Conditions - https://bccampus.ca/terms-conditions/
Privacy Policy - https://bccampus.ca/privacy/

**Comment by bdolor** [comment]
@murphybarb the footer, at the very bottom, is a menu item. It can be updated as a menu in the dashboard. 
As for the social media icons, I will put them in as they have been given to us so their largeness and whiteness can be on display so that other people besides myself to make a decision about them.

**Comment by murphybarb** [comment]
Hey @bdolor , thanks for clarifying about the bottom menu item. 

No need to place the social icons in the footer. I'm asking the designer to provide what you need so you just have to place what is provided, in the correct format. Thanks.

**Comment by murphybarb** [comment]
Hope this is helpful. If you need further details, please let me know. I can connect you with Jessica the designer or I can make the request for them. Thanks! 

From Jessica: 
I was picturing them being coded like these ones in the top right of this site https://victrans.com/ (an image + lots of css). It's a transparent image with a coloured background and width specified (saved at a larger size so its crisp on retina screens, with a width specified on the actual img src)

example:
<code>
<div class="social-icons">
   <a class="facebook" href="#"><img src="/images/icon-facebook.png" alt="Facebook" width="26"></a>
   <a class="googleplus" href="#"><img src="/images/icon-googleplus.png" alt="Google+" width="26"></a>
   <a class="twitter" href="#"><img src="/images/icon-twitter.png" alt="Twitter" width="26"></a>
</div>
</code>

The icons are to be round, rounded specified using css.

**Comment by murphybarb** [comment]
Ok, thanks. Requesting this from the designer. Standy by.

**Comment by murphybarb** [comment]
SVG files attached. 

[Basecamp Download(22).zip](https://github.com/BCcampus/bcc-sage/files/2354426/Basecamp.Download.22.zip)

**Comment by murphybarb** [comment]
Looks great. thanks!

**Comment by bdolor** [comment]
if you're satisfied, please move this and any other tickets to 'done'

**Comment by murphybarb** [comment]
Looks good! Couple of things: 

- I went into the dashboard was able to change the "quick links" to what they should be (that's awesome)! This is not a change request, just saying that's "awesome"!  

- I can't find where I can edit the "Subscribe to BCcampus News" text. It has to be edited and I understood we'd be able to edit this within the dashboard. Where do I access in the dashboard or do I have to provide text to you to update? 

- In the bottom part of the footer where "Terms and Conditions" and "Privacy" live, I've added the correct text and it is breaking weirdly. Would reducing the size of the text fix that?

**Item for design review:** padding and spacing around social icons and BCcampus logo.

**Comment by bdolor** [comment]
- colour of the links...from an accessibility and usability perspective, there are reasons to make links look different than regular text. I suppose you could argue it's in a menu area, but my preference would be to lean on these best practices. 
- subscribe to BCcampus News isn't currently editable - it's a block of html that's used in other areas of the site such as the news landing page, and in the side bar on various pages. We could open a new ticket to make the subscribe to bccampus news editable in the footer and the sidebar areas (via widgets) but I won't be able to commit to making it editable on the news landing page. 
- Terms and conditions have been given a bit more real estate on the footer so they don't have to wrap.

**Comment by murphybarb** [comment]
Re: the content for the newsletter subscription container/section. It's not essential we be able to edit it, so that's cool. **BUT** the content needs editing. Do I provide changes here or they can be handled later?

**Comment by murphybarb** [comment]
Change request to subscribe to the newsletter text in the footer: 

- **Change text** "Subscribe to BCcampus News" to "Subscribe to Our Newsletter"
- **Add punctuation** add a period at the end of the sentence "Get the latest information on news and events at BCcampus"
